frontend/src/components/pages/admin/CreateAssessmentPage.tsx

```diff
@@ -26,16 +35,36 @@ const CreateAssessmentPage = (): React.ReactElement => {
     register,
     formState: { errors },
     control,
-  } = useForm();
+  } = useForm<AssessmentData>();
 
   const location = useLocation();
+  const history = useHistory();
   const { date } = location.state as { date: string };
   const [validSubmit, setValidSubmit] = useState(true);
   const [assessmentName, setAssessmentName] = useState("");
+  const [createTest] = useMutation<{
+    createTest: { createTest: { id: string } };
+  }>(SAVE_ASSESSMENT);
 
-  const onSubmit = (data: any, e: any) => {
+  const onSubmit: SubmitHandler<AssessmentData> = async (data) => {
```

Comment:
i think you can just consume the type `TestRequest` instead of creating a new type

Reply:
the data type is slightly different than the TestRequest type! don't think it can be reused here